src/SymbolExpression.cpp

```diff
@@ -0,0 +1,74 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+ * @file SymbolExpression.cpp
+ * @date Fri February 25 2022
+ * @brief implementation for a symbol in a mathematical expression
+ */
+////////////////////////////////////////////////////////////////////////////////
+
+#include "SymbolExpression.h"
+
+#include <iostream>
+#include <memory>
+
+#include "ValueExpression.h"
+
+namespace calculator::expression
+{
+// ---------------------------------------------------------------------------//
+// //
+// SymbolExpression<T> Public Methods //
+// //
+// ---------------------------------------------------------------------------//
+template<typename T>
+SymbolExpression<T>::SymbolExpression(char glyph): glyph_{glyph}
+{
+}
+
+
```

Comment:
same on this page, just one space I believe, also if there are constructors/destructors, will need a comments